waku/node/v2/rpc/wakurpc.nim

```diff
@@ -40,6 +40,7 @@ proc setupWakuRPC*(node: WakuNode, rpcsrv: RpcServer) =
     proc handler(topic: string, data: seq[byte]) {.async, gcsafe.} =
       info "Hit subscribe handler", topic=topic, data=data
 
+    # TODO: Shouldn't we really be doing WakuNode subscribe here?
```

Comment:
Yeah we should, this is from before node API existed and wasn't updated correctly.